src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -200,7 +201,9 @@ FullyConnected_bf_tiled::GetAutoTuneParams(const fully_connected_params& params,
     while (max_tile_ofm * 2 * simd <= output_f && max_tile_ofm < 4)
         max_tile_ofm *= 2;
 
-    if (params.is_shape_agnostic) {
+    if (params.compressed && params.engineInfo.supports_immad) {
+        return selector.Default(tune_params(1, 1, 1, 4, 1, 1, EXE_MODE_DEFAULT));
```

Comment:
Just out of curiosity: are these parameters optimal even with large enough batch size?

Reply:
Maybe more tuning will be needed for batch > 1. For not it's targeting vector by matrix multiplication mostly